projects/novo-elements/src/elements/select/Select.spec.ts

```diff
@@ -35,306 +75,244 @@ xdescribe('Elements: NovoSelectElement', () => {
-    });
-    it('should not invoke onModelChange', () => {
-      jest.spyOn(comp, 'onModelChange');
-      comp.select({ value: 'foo' }, 1, false);
-      expect(comp.onModelChange).not.toHaveBeenCalled();
-    });
-    it('should not emit onSelect', () => {
-      jest.spyOn(comp.onSelect, 'emit');
-      comp.select({ value: 'foo' }, 1, false);
-      expect(comp.onSelect.emit).not.toHaveBeenCalled();
-    });
-  });
-
-  describe('Function: clear', () => {
-    it('should set selected', () => {
-      comp.placeholder = 'bar';
-      comp.selected = {
-        label: 'foo',
-        value: 'bar',
-        active: true,
-      };
-      comp.clear();
-      expect(comp.selected).toEqual({
-        label: 'bar',
-        value: null,
-        active: false,
-      });
-    });
-    it('should set active to false on previously selected object', () => {
-      const prevSelected = (comp.selected = {
-        label: 'foo',
-        value: 'bar',
-        active: true,
-      });
-      comp.clear();
-      expect(prevSelected.active).toEqual(false);
-    });
-    it('should set selectedIndex', () => {
-      comp.selectedIndex = 0;
-      comp.clear();
-      expect(comp.selectedIndex).toEqual(-1);
-    });
-    it('should set empty', () => {
-      comp.empty = false;
-      comp.clear();
-      expect(comp.empty).toEqual(true);
-    });
-  });
+  it('should propagate changes from NovoOption elements', fakeAsync(() => {
+    let selected;
+    comp.onSelect.subscribe(val => selected = val);
+    spyOn(comp, 'closePanel');
+    spyOn(comp, 'focus');
+    const options = [
+      { label: 'foo', value: 'foo' },
+      { label: 'bar', value: 'bar' },
+      { label: 'baz', value: 'baz' },
+    ];
+    fixture.componentRef.setInput('options', options);
+    fixture.detectChanges();
+    comp.writeValue('baz');
+    fixture.detectChanges();
+    expect(comp.viewOptions.length).toBe(3);
+    comp.openPanel();
+    const newNovoOption = comp.viewOptions.find(option => option.value === 'foo');
+    newNovoOption._selectViaInteraction();
+    tick();
+    expect(selected).toEqual({ selected: 'foo' });
+    expect((comp as HasOverlay).closePanel).toHaveBeenCalled();
+    expect(comp.focus).toHaveBeenCalled();
+  }));
 
   describe('Function: _handleKeydown(event)', () => {
-    xit('should not scroll', () => {});
     it('should close panel', () => {
       jest.spyOn(comp.overlay, 'closePanel');
       const mockEvent: any = { key: Key.Escape };
-      comp.header.open = true;
+      comp.openPanel();
       comp._handleKeydown(mockEvent);
       mockEvent.key = Key.Tab;
+      mockEvent.keyCode = TAB;
+      comp.openPanel();
       comp._handleKeydown(mockEvent);
       expect(comp.overlay.closePanel).toHaveBeenCalledTimes(2);
     });
-    it('should save header', () => {
-      const mockEvent: any = { key: Key.Enter };
+    // Possibly broken?
+    xit('should save header', () => {
```

Comment:
Did you track down if this is broken or not? It doesnt look like it was disabled before and we shouldn't merge in any disabled tests.